src/main/app/claims/ccjModelConverter.ts

```diff
@@ -37,13 +38,23 @@ function convertPayBySetDate (draftCcj: DraftCCJ): Moment {
 export class CCJModelConverter {
 
   static convert (draftCcj: DraftCCJ): CountyCourtJudgment {
+    let statementOfTruth: StatementOfTruth = undefined
+    if (draftCcj.qualifiedDeclaration) {
+      statementOfTruth = new StatementOfTruth(
```

Comment:
This was slightly confusing for me. What do you think about renaming this Claim Store model class into something more generic like e.g. `QualifiedSignature`? Or maybe adding a comment?

Reply:
will add a comment for now.